persistence/src/test/scala/hmda/persistence/processing/TestConfigOverride.scala

```diff
@@ -10,6 +10,29 @@ object TestConfigOverride {
       | akka.persistence.journal.plugin = "inmemory-journal"
       | akka.persistence.query.journal.id = "inmemory-read-journal"
       | akka.persistence.snapshot-store.plugin = "inmemory-snapshot-store"
+      | akka.actor {
```

Comment:
I'm interested in the difference between this Override test config and the normal test config (in `persistence/src/test/resources/application.conf`). 

What is the difference between the tests that have to use the override, and the ones that don't?